softserve/__init__.py

```diff
@@ -20,4 +20,4 @@
 from views import about  # noqa: E402, F401
 from model import User, NodeRequest, Vm  # noqa: E402, F401
 
-db.create_all()
+# db.create_all()
```

Comment:
Remove this line entirely. You should not be using `db.create_all()` anymore, especially after you the migration to do this.